index.html

```diff
@@ -11,9 +11,223 @@
   <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.3/css/all.css" 
   integrity="sha384-SZXxX4whJ79/gErwcOYf+zWLeJdY/qpuqC4cAa9rOGUstPomtqpuNWT9wdPEn2fk" 
   crossorigin="anonymous">
+  <link rel="stylesheet" href="style.css">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" 
+  integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" 
+  crossorigin="anonymous">
   <title>Psych Conf</title>
 </head>
 <body>
-  
+  <main>
+    <header>
+      <div class="topnav" id="myTopnav">
+        <img class="img-responsive logo" 
+        src="images/psych-logo.png" alt="Psych Conference Logo" />
```

Comment:
- [ ] You mentioned in the PR summary that the logo in the header links to the home page. This does not seem to be true. Kindly make the logo a link so that it links to the homepage :blush:

Reply:
Great job! 💯